staging-service/src/main/java/com/github/cafdataprocessing/services/staging/StagingController.java

```diff
@@ -179,13 +179,13 @@ public ResponseEntity<BatchList> getBatches(
             batchList.entries(batchFiles);
             return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(batchList);
         } catch (final InvalidTenantIdException ex) {
-            LOGGER.error("Invalid X-TENANT-ID ", ex);
+            LOGGER.warn("Invalid X-TENANT-ID {}", X_TENANT_ID);
             throw new WebMvcHandledRuntimeException(HttpStatus.BAD_REQUEST, ex.getMessage());
         } catch (final InvalidBatchIdException ex) {
-            LOGGER.error("Invalid batchId ", ex);
+            LOGGER.warn("Invalid batchId {}", ex.getMessage());
             throw new WebMvcHandledRuntimeException(HttpStatus.BAD_REQUEST, ex.getMessage());
         } catch (final StagingException ex) {
-            LOGGER.error("Error in getBatches ", ex);
+            LOGGER.warn("Error in getBatches {}", ex.getMessage());
```

Comment:
Are we sure the stack trace would not be useful in this case?

Reply:
I think it would be. Putting it back.

Reply:
This doesn't seem right, Dermot is asking for stack trace, I don't believe the message contains that.

Reply:
Ya I meant I have reverted my change of getting message from exception object. Stack trace would be printed like before.